Buildkite/ephemeral-agent.md

```diff
@@ -0,0 +1,107 @@
+2. Copy the provided scripts to the agent machine: [base.sh](scripts/base.sh), [bootstrap.sh](scripts/bootstrap.sh), [run.sh](scripts/run.sh). 
+**Note**: All scripts should be in the same directory. For example you can add them to `/usr/local/var/buildkite-agent/`. Overwrite the existing `bootstrap.sh` file if asked.
+3. Verify the scripts can be executed by running `chmod +x path_to_script` in the command line.
+4. Copy the provided [pre-exit hook](scripts/pre-exit) to the agent machine. You should add it to the agent hooks directory. It can be found in the `buildkite-agent.cfg` file under the `hooks-path` property. The location of the `buildkite-agent.cfg` varies depending on your OS. For more information, see your [platform's installation article][agent-instructions].
+5. Verify the hook can be executed by running `chmod +x path_to_hook` in the command line.
+6. Verify [jq][jq] is installed. For more information, see [jq][jq] page.
+7. Verify the machine has network visibility to the Orka environment. If the machine is part of the Orka environment, skip this step. You can use any VPN client to connect to the Orka environment. For more information, see your Orka [IP Plan][ip-plan]
+
+## Buildkite environment variables
+
+The provided scripts expect the following environment variables to be set:
+
+* `ORKA_USER` - User used to connect to the Orka environment. Created by running `orka user create`
+* `ORKA_PASSWORD` - Password used to connect to the Orka environment. Created by running `orka user create`
+* `ORKA_ENDPOINT` - The Orka endpoint. By default it is `http:/10.10.10.100`
+* `ORKA_VM_NAME` - The name of the VM to be deployed. This should match the VM config created [earlier](#set-up-an-orka-vm-config)
+* `ORKA_VM_USER` - User used to SSH to the VM
+
+By default the wrapper agent and the ephemeral agent share the same Buildkite specific environment variables.  
+This means that the wrapper agent and the ephemeral agent will have the same `hook-path` for example.
+If you wish to use different environment variables on the ephemeral agent you can set the following optional environment variables:
+
+* `BUILDKITE_AGENT_ACCESS_TOKEN_SUBAGENT`
+* `BUILDKITE_BUILD_PATH_SUBAGENT`
+* `BUILDKITE_HOOKS_PATH_SUBAGENT`
+* `BUILDKITE_PLUGINS_PATH_SUBAGENT`
+* `BUILDKITE_PLUGINS_ENABLED_SUBAGENT`
+* `BUILDKITE_PLUGINS_ENABLED_SUBAGENT`
+* `BUILDKITE_PLUGIN_VALIDATION_SUBAGENT`
+* `BUILDKITE_LOCAL_HOOKS_ENABLED_SUBAGENT`
+* `BUILDKITE_SSH_KEYSCAN_SUBAGENT`
+* `BUILDKITE_AGENT_DEBUG_SUBAGENT`
+
+All of the above override the Buildkite environment variables with the same name without the `_SUBAGENT` suffix.
+For example: `BUILDKITE_AGENT_ACCESS_TOKEN_SUBAGENT` overwrites `BUILDKITE_AGENT_ACCESS_TOKEN`.
+
+For more information about Buildkite environment variables, see [here][env-variables].
+
+## Using the Buildkite agent
+
+Once the setup of the Buildkite agent is finished, you can run your CI/CD pipelines in Orka.
```

Comment:
the wrapper agent or the ephemeral agent :P

Reply:
You do not set up the ephemeral, so the wrapper :)

Reply:
Keep in mind that users might not read this thoroughly and might just scan through the doc, this way missing out on some of the context. I recommend always using "wrapper" and "ephemeral" along with agent in the text to avoid any possible confusion.